.github/workflows/codeql.yml

```diff
@@ -0,0 +1,52 @@
+name: "Code scanning - action"
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 19 * * 0'
```

Comment:
cron might not be necessary. push and PR events should be sufficient

Reply:
You're right!
it was a test and I forgot to restore

Reply:
can you remove it?

Reply:
Of course 
I'm doing

Reply:
Done!!
Regards